src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx

```diff
@@ -99,6 +116,20 @@ const PublicationDetailPage = () => {
                     {body && <CustomHTMLBlock body={body} />}
                   </EditorialContent>
                 </Column>
+
+                {related && related.length ? (
```

Comment:
You could use optional chaining here.
```suggestion
                {related?.length ? (
```

Reply:
Good one, done!